packages/relay/src/lib/precheck.ts

```diff
@@ -38,8 +38,14 @@ export class Precheck {
     this.logger = logger;
   }
 
-  value(transaction: string) {
-    const tx = ethers.utils.parseTransaction(transaction);
+  private static parseTxIfNeeded(transaction: string | ethers.Transaction): ethers.Transaction {
+    return typeof transaction === 'string'
```

Comment:
Not sure if string type is needed with use of helper methods.
Just make all prechecks take in `Transaction` unless that make testing more difficult.